metadata-ingestion/src/datahub/ingestion/source/unity/usage.py

```diff
@@ -100,12 +105,14 @@ def _generate_workunits(self, table_map: TableMap) -> Iterable[MetadataWorkUnit]
             )
             return
 
-        for wu in self.usage_aggregator.generate_workunits(
-            resource_urn_builder=self.table_urn_builder,
-            user_urn_builder=self.user_urn_builder,
-        ):
-            self.report.num_usage_workunits_emitted += 1
-            yield wu
+        yield from auto_empty_dataset_usage_statistics(
```

Comment:
noticed that the call to `auto_empty_dataset_usage_statistics` is in `get_usage_workunits` in some sources whereas in `_get_workunits_internal` in some sources. Would be good to keep it consistent.

Reply:
This is due to inconsistency in how the `include_usage_stats` param is interpreted. For snowflake, if `include_usage_stats` is false you can still ingest operational stats, while for bigquery, redshift, and unity, if `include_usage_statistics` is false then we don't ingest usage at all. We should standardize here, but idk which is best